### ISSUE TITLE
/list command not working

### ISSUE THREAD

**Issue by leeann-chu** [RayNieport/mconBot#6]
First off, thank you so much for creating this bot! It was exactly what I was looking for to remotely control my mc server. 
However, although commands like "start, stop, weather clear, and say" all work perfectly... "list" always returns back 

![image](https://user-images.githubusercontent.com/80134915/169671077-b704ec17-8365-488b-a35e-e0193ec357c0.png)

Even when there are definitely players online. 

Do you ever have this problem? When I do the command in the server terminal it works exactly how you'd expect. 
![image](https://user-images.githubusercontent.com/80134915/169671155-d32a174c-b325-4f6c-ad40-06e737ac8df7.png)

So I'm confused why rcon doesn't see them. I have the repo for my bot set to public if you want to see how I changed your code. [here](https://github.com/leeann-chu/pfa-automaton)

Extra info: I'm not running this in a docker and I'm running a forge 1.7.10 modded server.

**Comment by RayNieport** [comment]
Hi! Glad you're enjoying the bot. I'm running a Spigot server and use `list` pretty frequently. The output looks like this for me:

![image](https://user-images.githubusercontent.com/83410313/169671579-f7c02b5c-1e6f-49aa-8ac6-187a363cd163.png)

From the server terminal, I get:

![image](https://user-images.githubusercontent.com/83410313/169671619-dc8e44db-1396-4248-93b5-e8f14b157056.png)

If you could, please try using a third-party RCON client to test the command and let me know the results. My bot should simply return whatever RCON gets from the server. It could be possible that your specific forge server has a bug when using RCON for this command. I just want to check that possibility before delving into your modified code. Thanks!

**Comment by leeann-chu** [comment]
Alright, I downloaded [this](https://github.com/Tiiffi/mcrcon) third party RCON client. 
Here are my results:  

![image](https://user-images.githubusercontent.com/80134915/169672296-bfbeee63-dc46-4eaa-bb64-c03d1e1fa58e.png)

The first command "say" worked, but you can see how it still insists no one is online. After this test I'm starting to think either 
a) 1.7.10 had a different \list command output or b) my forge server is intercepting the list command somehow, though I took a look at all the mods and none stick out as ones that add or overwrite vanilla commands. Any thoughts?

**Comment by RayNieport** [comment]
Looking at the history of the `list` command: [https://minecraft.fandom.com/wiki/Commands/list](https://minecraft.fandom.com/wiki/Commands/list) there does not seem to have been a modification to the command syntax in newer versions.

My best guess is your Spigot server is somehow interfering. I would try out a vanilla 1.7.10 server and see what happens.

**Comment by RayNieport** [comment]
Looks like it works fine in a 1.7.10 vanilla server.

**Comment by leeann-chu** [comment]
oh I was just in the middle of testing that! Can confirm, vanilla server works for me too. Alright, now to test forge!

**Comment by leeann-chu** [comment]
Alright 1.7.10 forge works for me somehow... so it must be a mod on my other server that's doing this. Thanks for all your help, I'm gonna close the issue now cuz it's clearly 100% on my end. 

Have a nice day! Thank you.

**Comment by RayNieport** [comment]
Yeah I just checked standard 1.7.10 Forge as well and it works no problem. Must be a mod you're using.

Hope you get it working! Glad you're enjoying the bot. Let me know if there's anything else I can help with.
So cool to see people using my code and modifying to make it their own. Have fun!

**Comment by leeann-chu** [comment]
Turns out it it's the mod FTBLibrary in the slim case anyone else has this problem and wants to know. 
I don't really want to remove it cuz it's the core mod that allows chunk claiming/loading for farms. 

Funny enough, they were also the first mods I googled to see their [list of commands](https://help.ftb.team/help/en-gb/8-frequently-asked-questions/20-is-there-a-full-list-of-commands). But \list is not mentioned. 

Unfortunately, I think I'm at a dead end. As you said, this is 2014 stuff, so if it works weird in a really niche case that's just how it is. On the bright side, I at least know _why_ it doesn't work now.
